x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/create_jobs.ts

```diff
@@ -27,31 +26,19 @@ export async function createJobs({
   toasts: NotificationsStart['toasts'];
 }) {
   try {
-    const res = await callApmApi({
+    await callApmApi({
       pathname: '/api/apm/settings/anomaly-detection/jobs',
       method: 'POST',
       params: {
         body: { environments },
       },
     });
 
-    // a known error occurred
-    if (res?.errorCode) {
-      toasts.addDanger({
-        title: errorToastTitle,
-        text: MLErrorMessages[res.errorCode],
-      });
-      return false;
-    }
```

Comment:
Got rid of the `errorCode` in the response. Now the server side will throw the error and the client should simply read the message.